java/code/src/com/suse/manager/webui/utils/MinionActionUtils.java

```diff
@@ -73,7 +63,7 @@
     private static final Logger LOG = Logger.getLogger(MinionActionUtils.class);
 
     /** Whether the current database is Postgres. */
-    public static final boolean POSTGRES = ConfigDefaults.get().isPostgresql();
+    public static final boolean POSTGRES = true;
```

Comment:
Is this still somewhere used or can we remove that constant as well?

Reply:
Oh yes, it should be removed as well.